solution/ai/notebooks/YOLOV8/rock-paper-scissors.ipynb

```diff
@@ -63,9 +63,9 @@
    "outputs": [],
    "source": [
     "# go to the folder corresponding to your object container\n",
-    "%cd /workspace/attendee\n",
+    "%cd /workspace/data\n",
     "\n",
-    "!curl -L \"https://storage.gra.cloud.ovh.net/v1/AUTH_2b59f4f857964cc28bb66c7155b6c8d9/workshop-resources/Rock-Paper-Scissors-SXSW.v12i.yolov8.zip\" > roboflow.zip; unzip roboflow.zip; rm roboflow.zip"
+    "!curl -L \"<dataset_url>\" > roboflow.zip; unzip roboflow.zip; rm roboflow.zip"
```

Comment:
Is it possible to use a variable for this <dataset_url>?

Reply:
Pas possible de variabliser le notebook à son lancement mais je peux essayer de rajouter une variable dans la cellule pour que ce soit le contenu de la variable à modifier et non l'URL dans le curl.

Reply:
fix by https://github.com/devrel-workshop/101-AI-and-py/pull/85/commits/a741f73957d3452ef9e28eab95b41b194a4da2c7

@titimoby I know it's not what you had in mind, but I think it's the best I can do to variabilize the URL 🤷‍♂️